cmd/launcher/control.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"context"
+
+	"github.com/boltdb/bolt"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/kit/actor"
+	"github.com/kolide/launcher/pkg/control"
+	kolideLog "github.com/kolide/launcher/pkg/log"
+	"github.com/pkg/errors"
+)
+
+func createControl(ctx context.Context, db *bolt.DB, logger *kolideLog.Logger, opts *options) (*actor.Actor, error) {
```

Comment:
IMO this should accept a `github.com/go-kit/kit/log.Logger`. The swap logger functionality was abstracted into `github.com/kolide/kit`: https://github.com/kolide/kit/blob/45f7c9242522e5dd787e33bdec670fd8aec4ed60/logutil/logutil.go#L45